oscarlab/graphene
# [Pal] Prefix globals with g_ (part 1)

## Description of the changes 

A small step in fixing the old code to look reasonably.

Only a part of PAL globals is fixed in this PR to reduce conflicts with open PRs.

## How to test this PR? 

Jenkins + grep to see if I didn't miss any occurrences of renamed variables.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1645)
